components/scream/tests/coupled/shoc_cld_spa_p3_rrtmgp/CMakeLists.txt

```diff
@@ -1,12 +1,22 @@
 INCLUDE (ScreamUtils)
 
 # Create the test
-SET (TEST_LABELS "shoc;cld;p3;rrtmgp;physics")
-SET (NEED_LIBS shoc cld_fraction p3 scream_rrtmgp scream_control scream_share physics_share)
-CreateUnitTest(shoc_cld_p3_rrtmgp shoc_cld_p3_rrtmgp.cpp "${NEED_LIBS}" LABELS ${TEST_LABELS}
+SET (TEST_LABELS "shoc;cld;spa;p3;rrtmgp;physics")
+SET (NEED_LIBS shoc cld_fraction spa p3 scream_rrtmgp scream_control scream_share physics_share)
+CreateUnitTest(shoc_cld_spa_p3_rrtmgp shoc_cld_spa_p3_rrtmgp.cpp "${NEED_LIBS}" LABELS ${TEST_LABELS}
   MPI_RANKS 1 ${SCREAM_TEST_MAX_RANKS}
-  PROPERTIES FIXTURES_SETUP shoc_cld_p3_rrmtgp_generate_output_nc_files
+  PROPERTIES FIXTURES_SETUP shoc_cld_spa_p3_rrmtgp_generate_output_nc_files
 )
+# Make sure the SPA data directory is correctly defined
+if (NOT EXISTS ${SCREAM_SPA_DATA_DIR})
```

Comment:
Same here: a user without spa data folder cannot run coupled physics test. We should have a full physics test without spa.